Library/OcConsoleLib/TextOutputBuiltin.c

```diff
@@ -937,38 +946,6 @@ EFI_CONSOLE_CONTROL_PROTOCOL
   ConsoleControlLockStdIn
 };
 
-VOID
-ConsoleControlInstall (
-  VOID
-  )
-{
-  EFI_STATUS                    Status;
-  EFI_CONSOLE_CONTROL_PROTOCOL  *ConsoleControl;
-
-  Status = OcHandleProtocolFallback (
-             &gST->ConsoleOutHandle,
-             &gEfiConsoleControlProtocolGuid,
-             (VOID *)&ConsoleControl
-             );
-
-  if (!EFI_ERROR (Status)) {
-    ConsoleControl->SetMode (ConsoleControl, EfiConsoleControlScreenGraphics);
-
-    CopyMem (
-      ConsoleControl,
-      &mConsoleControlProtocol,
-      sizeof (mConsoleControlProtocol)
-      );
-  }
-
-  gBS->InstallMultipleProtocolInterfaces (
-         &gST->ConsoleOutHandle,
-         &gEfiConsoleControlProtocolGuid,
-         &mConsoleControlProtocol,
-         NULL
-         );
-}
-
```

Comment:
I believe this removed, unused code is possibly a latent genetic memory of having faced the same issue which this PR addresses before.